README.md

```diff
@@ -123,7 +123,7 @@ Our community also has:
 - A [forum](https://www.freecodecamp.org/forum) where you can usually get programming help or project feedback within hours.
 - A [YouTube channel](https://youtube.com/freecodecamp) with free courses on Python, SQL, Android, and a wide variety of other technologies.
 - A [podcast](https://podcast.freecodecamp.org/) with technology insights and inspiring stories from developers.
-- A comprehensive [guide to thousands of programming topics](https://guide.freecodecamp.org/)
+- A comprehensive [guide to thousands of programming topics](https://guide.freecodecamp.org/) (Archived and no longer updated, but still a great resource)
```

Comment:
Actually, this line can be totally removed as it no longer exists in the repo after the next release.
```suggestion
```

Reply:
Well, that makes for a much cleaner solution then. Though if the archived site will still be up at all it may be nice to still have a link to it somewhere accessible as that was the only link to the web page I could find. Regardless, it doesn't make sense to have it lingering at the bottom of this repo, so I agree.